Convert to !include paradigm

lewisgross1296 opened the pull request:
Closes #859.

Pretty sure this will not affect any of the gold files' results. LMK if you want me to run them, though.

lewisgross1296 commented:
Thoughts on removing the 
```
# copy-pasta from common.i
```
from the tutorials? I think it's there to help the reader, but it is a little redundant IMO.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2338104/) on 52b3b00 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/cardinal/docs/PRs/935/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/2489489/), step Sync to remote on 3c7635e wanted to post the following:

View the site [here](https://mooseframework.inl.gov/cardinal/docs/PRs/935/site)

This comment will be updated on new commits.

nuclearkevin commented:
@lewisgross1296 I believe you'll need to modify `tutorials/gas_compact_multiphysics/tests` to get rid of the accidental input file merging. Swapping `openmc_thm_moose` and `openmc_nek_moose` to the following should get the tests passing:
```
[openmc_thm_moose]
  type = RunApp
  input = openmc_thm.i
  prereq = gen_mesh
  mesh_mode = 'replicated' # MultiAppGeometricInterpolationTransfer not supported
  cli_args = Problem/particles=200 Executioner/num_steps=2 Problem/inactive_batches=10 Problem/batches=20 Problem/Tallies/heat_source/trigger=none Problem/k_trigger=none Outputs/csv/file_base=openmc_thm_test_out Problem/identical_cell_fills="compacts compacts_trimmer_tri"'
  min_parallel = 2
  requirement = 'The system shall solve for coupled OpenMC, heat conduction, and THM fluid flow for a TRISO compact.'
  required_objects = 'OpenMCCellAverageProblem'
[]
[openmc_nek_moose] # can't test the nek.i, too big
  type = RunApp
  input = openmc_nek.i
  prereq = gen_mesh
  min_parallel = 2
  mesh_mode = 'replicated' # MultiAppGeometricInterpolationTransfer not supported
  cli_args = "Problem/particles=200 Executioner/num_steps=1 Problem/inactive_batches=10 Problem/batches=20 Problem/Tallies/heat_source/trigger=none Problem/k_trigger=none Outputs/csv/file_base=openmc_nek_test_out Problem/identical_cell_fills='compacts compacts_trimmer_tri' bison:MultiApps/active='' bison:Transfers/active='' Problem/initial_properties=xml"
  requirement = 'The system shall solve for coupled OpenMC and heat conduction for a TRISO compact.'
  required_objects = 'OpenMCCellAverageProblem'
[]
```

lewisgross1296 commented:
Thanks for the tip @nuclearkevin! I think `gas_assembly/tests` probably also needs the `input` modification too. I will modify and push

lewisgross1296 commented:
Just realized there's some more "copy-pasta" lines in some nek iniput files. Does nek ignore the duplicate definitions that MOOSE typically tags as an error? I can remove them for consistency if desired, @aprilnovak.

Also, seems like the pre-check had issues with my branch after I modified some `tests`. Is this this something I need to fix/if so, how?
```

BUILD_ROOT/: Cloning https://github.com/neams-th-coe/cardinal.git
BUILD_ROOT/: git clone -q --dissociate --reference /mnt/civet/gitrefs/github.com/neams-th-coe/cardinal https://github.com/neams-th-coe/cardinal.git
BUILD_ROOT/cardinal/: Checking out neams-th-coe/cardinal:devel for testing
BUILD_ROOT/cardinal/: git checkout -b test origin/devel
Switched to a new branch 'test'
branch 'test' set up to track 'origin/devel'.
BUILD_ROOT/cardinal/: Merging lewisgross1296/cardinal:include_paradigm into neams-th-coe/cardinal:devel
BUILD_ROOT/cardinal/: git pull --no-ff https://github.com/lewisgross1296/cardinal.git include_paradigm
From https://github.com/lewisgross1296/cardinal
 * branch              include_paradigm -> FETCH_HEAD
Auto-merging doc/content/tutorials/openmc_fluid.md
Auto-merging doc/content/tutorials/triso_multiphysics.md
Auto-merging tutorials/gas_assembly/openmc.i
Auto-merging tutorials/gas_compact_multiphysics/openmc_nek.i
Auto-merging tutorials/gas_compact_multiphysics/openmc_thm.i
Auto-merging tutorials/gas_compact_multiphysics/tests
CONFLICT (content): Merge conflict in tutorials/gas_compact_multiphysics/tests
Automatic merge failed; fix conflicts and then commit the result.
```

aprilnovak commented:
You have a merge conflict -- you should rebase this branch, fix the conflict, and then force push.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/2496514/), step Generate coverage on 569cbfa wanted to post the following:


## Coverage

Coverage did not change

[Full coverage report](https://mooseframework.inl.gov/cardinal/docs/PRs/935/coverage)



This comment will be updated on new commits.
